src/App.js

```diff
@@ -1,19 +1,50 @@
 import React from 'react';
 import './App.css';
+import { useState } from 'react';
 import chatMessages from './data/messages.json';
+import ChatLog from './components/ChatLog';
+
 
 const App = () => {
+  const[chatData, setChatData] = useState(chatMessages);
+
+  const calcTotalLikes = (chatData) => {
+    return chatData.reduce((total, message) => {
```

Comment:
Nice! I love that you all used the `reduce` built-in function for calculating the total number of likes! Makes your code more concise!